### ISSUE TITLE
FFmpegPostProcessorError

### ISSUE THREAD

**Issue by nicolaasjan** [yt-dlp/yt-dlp#6517]
### DO NOT REMOVE OR SKIP THE ISSUE TEMPLATE

- [X] I understand that I will be **blocked** if I *intentionally* remove or skip any mandatory\* field

### Checklist

- [X] I'm reporting that a **supported** site is broken
- [X] I've verified that I'm running yt-dlp version **2023.03.04** ([update instructions](https://github.com/yt-dlp/yt-dlp#update)) or later (specify commit)
- [X] I've checked that all provided URLs are playable in a browser with the same IP and same login details
- [X] I've checked that all URLs and arguments with special characters are [properly quoted or escaped](https://github.com/yt-dlp/yt-dlp/wiki/FAQ#video-url-contains-an-ampersand--and-im-getting-some-strange-output-1-2839-or-v-is-not-recognized-as-an-internal-or-external-command)
- [X] I've searched [known issues](https://github.com/yt-dlp/yt-dlp/issues/3766) and the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues **including closed ones**. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)
- [X] I've read about [sharing account credentials](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#are-you-willing-to-share-account-details-if-needed) and I'm willing to share it if required

### Region

The Netherlands

### Provide a description that is worded well enough to be understood

`yt-dlp version nightly@2023.03.11 [2a23d92d9] (zip)`

Requesting **-f 299+140** on any video (in this case **L7z-VeAhwCw**), gives me 2 unmerged files:
`Planets Can Survive Being Swallowed by a Star With Surprising Effects [L7z-VeAhwCw].f140.m4a` 
and
`Planets Can Survive Being Swallowed by a Star With Surprising Effects [L7z-VeAhwCw].f299.mp4`

The "m4a" file is in fact a **1280x720** VP9 @230 kb/s **webm** file with no sound.
The "mp4" file is a **1280x720** VP9 @618 kb/s **webm**  file with no sound.

MediaInfo says:
`FileExtension_Invalid                    : webm`

Expected result:
1920x1080 mp4

(I omitted `-U` in verbose below, because this is already the latest now)

### Provide verbose output that clearly demonstrates the problem

- [X] Run **your** yt-dlp command with **-vU** flag added (`yt-dlp -vU <your command line>`)
- [ ] If using API, add `'verbose': True` to `YoutubeDL` params instead
- [X] Copy the WHOLE output (starting with `[debug] Command-line config`) and insert it below

### Complete Verbose Output

```shell
yt-dlp --ignore-config -f 299+140 -v https://www.youtube.com/watch?v=L7z-VeAhwCw
[debug] Command-line config: ['--ignore-config', '-f', '299+140', '-v', 'https://www.youtube.com/watch?v=L7z-VeAhwCw']
[debug] Encodings: locale UTF-8, fs utf-8, pref UTF-8, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version nightly@2023.03.11 [2a23d92d9] (zip)
[debug] Python 3.8.10 (CPython x86_64 64bit) - Linux-5.4.0-144-generic-x86_64-with-glibc2.29 (OpenSSL 1.1.1f  31 Mar 2020, glibc 2.31)
[debug] exe versions: ffmpeg N-109989-g7a8560cb22-20230310 (setts), ffprobe N-109989-g7a8560cb22-20230310, phantomjs 2.1.1, rtmpdump 2.4
[debug] Optional libraries: Cryptodome-3.17, brotli-1.0.9, certifi-2022.12.07, mutagen-1.46.0, secretstorage-3.3.3, sqlite3-2.6.0, websockets-10.4, xattr-0.9.6
[debug] Proxy map: {}
[debug] Loaded 1788 extractors
[youtube] Extracting URL: https://www.youtube.com/watch?v=L7z-VeAhwCw
[youtube] L7z-VeAhwCw: Downloading webpage
[youtube] L7z-VeAhwCw: Downloading android player API JSON
[debug] Loading youtube-nsig.21246a91 from cache
[debug] [youtube] Decrypted nsig LlE2gfofrb3kiVoYF => g-IZ1y8t_gfAmQ
[debug] Sort order given by extractor: quality, res, fps, hdr:12, source, vcodec:vp9.2, channels, acodec, lang, proto
[debug] Formats sorted by: hasvid, ie_pref, quality, res, fps, hdr:12(7), source, vcodec:vp9.2(10), channels, acodec, lang, proto, filesize, fs_approx, tbr, vbr, abr, asr, vext, aext, hasaud, id
[info] L7z-VeAhwCw: Downloading 1 format(s): 299+140
[debug] Invoking dashsegments downloader on "https://rr4---sn-5hne6nzy.googlevideo.com/videoplayback?expire=1678630780&ei=HIsNZIrpF4-5x_APkqe_6AE&ip=<REDACTED>&id=o-ALDz-H97XsaOe1t48pRlsES4obCSScJjDI30D8d6dA2W&itag=299&source=youtube&requiressl=yes&mh=4a&mm=31%2C26&mn=sn-5hne6nzy%2Csn-4g5edndl&ms=au%2Conr&mv=m&mvi=4&pl=17&initcwndbps=2936250&spc=H3gIhhtlHgdlhgehy-10__JkcIzDEDQ&vprv=1&svpuc=1&mime=video%2Fmp4&gir=yes&clen=201632976&dur=680.466&lmt=1678580550165585&mt=1678608777&fvip=1&keepalive=yes&fexp=24007246&c=ANDROID&txp=3319224&sparams=expire%2Cei%2Cip%2Cid%2Citag%2Csource%2Crequiressl%2Cspc%2Cvprv%2Csvpuc%2Cmime%2Cgir%2Cclen%2Cdur%2Clmt&sig=AOq0QJ8wRQIhAPJHavk7NBp1-dDVfm8RYz-6BSSKgoxQejLDvsNqrFs4AiBTngoHV-NkuLJgV7a8rgvXItA1sipYIDsyw1rnOZNtYw%3D%3D&lsparams=mh%2Cmm%2Cmn%2Cms%2Cmv%2Cmvi%2Cpl%2Cinitcwndbps&lsig=AG3C_xAwRAIgNhzkaHNzyx4KGP-fFKJkS5h1AjZ9U3q1yFrqs8JdviUCIGxFiJGW86stgdT3DMLmx5rfOuOdOAhW7o0JXBN0WpuV"
[dashsegments] Total fragments: 20
[download] Destination: Planets Can Survive Being Swallowed by a Star With Surprising Effects [L7z-VeAhwCw].f299.mp4
[download]  30.0% of ~ 166.99MiB at    1.15MiB/s ETA 00:11 (frag 6/20)[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 7 (1/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 7 (2/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 7 (3/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 7 (4/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 7 (5/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 7 (6/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 7 (7/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 7 (8/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 7 (9/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 7 (10/10)...
[download] fragment not found; Skipping fragment 7 ...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 8 (1/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 8 (2/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 8 (3/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 8 (4/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 8 (5/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 8 (6/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 8 (7/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 8 (8/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 8 (9/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 8 (10/10)...
[download] fragment not found; Skipping fragment 8 ...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 9 (1/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 9 (2/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 9 (3/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 9 (4/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 9 (5/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 9 (6/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 9 (7/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 9 (8/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 9 (9/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 9 (10/10)...
[download] fragment not found; Skipping fragment 9 ...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 10 (1/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 10 (2/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 10 (3/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 10 (4/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 10 (5/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 10 (6/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 10 (7/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 10 (8/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 10 (9/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 10 (10/10)...
[download] fragment not found; Skipping fragment 10 ...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 11 (1/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 11 (2/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 11 (3/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 11 (4/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 11 (5/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 11 (6/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 11 (7/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 11 (8/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 11 (9/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 11 (10/10)...
[download] fragment not found; Skipping fragment 11 ...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 12 (1/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 12 (2/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 12 (3/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 12 (4/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 12 (5/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 12 (6/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 12 (7/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 12 (8/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 12 (9/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 12 (10/10)...
[download] fragment not found; Skipping fragment 12 ...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 13 (1/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 13 (2/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 13 (3/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 13 (4/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 13 (5/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 13 (6/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 13 (7/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 13 (8/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 13 (9/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 13 (10/10)...
[download] fragment not found; Skipping fragment 13 ...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 14 (1/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 14 (2/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 14 (3/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 14 (4/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 14 (5/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 14 (6/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 14 (7/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 14 (8/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 14 (9/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 14 (10/10)...
[download] fragment not found; Skipping fragment 14 ...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 15 (1/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 15 (2/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 15 (3/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 15 (4/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 15 (5/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 15 (6/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 15 (7/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 15 (8/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 15 (9/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 15 (10/10)...
[download] fragment not found; Skipping fragment 15 ...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 16 (1/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 16 (2/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 16 (3/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 16 (4/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 16 (5/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 16 (6/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 16 (7/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 16 (8/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 16 (9/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 16 (10/10)...
[download] fragment not found; Skipping fragment 16 ...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 17 (1/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 17 (2/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 17 (3/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 17 (4/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 17 (5/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 17 (6/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 17 (7/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 17 (8/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 17 (9/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 17 (10/10)...
[download] fragment not found; Skipping fragment 17 ...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 18 (1/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 18 (2/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 18 (3/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 18 (4/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 18 (5/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 18 (6/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 18 (7/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 18 (8/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 18 (9/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 18 (10/10)...
[download] fragment not found; Skipping fragment 18 ...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 19 (1/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 19 (2/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 19 (3/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 19 (4/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 19 (5/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 19 (6/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 19 (7/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 19 (8/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 19 (9/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 19 (10/10)...
[download] fragment not found; Skipping fragment 19 ...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 20 (1/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 20 (2/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 20 (3/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 20 (4/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 20 (5/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 20 (6/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 20 (7/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 20 (8/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 20 (9/10)...
[download] Got error: HTTP Error 400: Bad Request. Retrying fragment 20 (10/10)...
[download] fragment not found; Skipping fragment 20 ...
[download] 100% of   50.10MiB in 00:00:11 at 4.37MiB/s
[debug] Invoking dashsegments downloader on "https://rr4---sn-5hne6nzy.googlevideo.com/videoplayback?expire=1678630780&ei=HIsNZIrpF4-5x_APkqe_6AE&ip=<REDACTED>&id=o-ALDz-H97XsaOe1t48pRlsES4obCSScJjDI30D8d6dA2W&itag=140&source=youtube&requiressl=yes&mh=4a&mm=31%2C26&mn=sn-5hne6nzy%2Csn-4g5edndl&ms=au%2Conr&mv=m&mvi=4&pl=17&initcwndbps=2936250&spc=H3gIhhtlHgdlhgehy-10__JkcIzDEDQ&vprv=1&svpuc=1&mime=audio%2Fmp4&gir=yes&clen=11014317&dur=680.507&lmt=1678580714674064&mt=1678608777&fvip=1&keepalive=yes&fexp=24007246&c=ANDROID&txp=3318224&sparams=expire%2Cei%2Cip%2Cid%2Citag%2Csource%2Crequiressl%2Cspc%2Cvprv%2Csvpuc%2Cmime%2Cgir%2Cclen%2Cdur%2Clmt&sig=AOq0QJ8wRQIgE5U1ddc7RfmdUgi4kiVKMuEW3-BBtVfibL_BN2SxsJMCIQDqJXN_ieJqYXhwPVLn-AYyXBp0C4jnQQJTV1lVqDyEvw%3D%3D&lsparams=mh%2Cmm%2Cmn%2Cms%2Cmv%2Cmvi%2Cpl%2Cinitcwndbps&lsig=AG3C_xAwRAIgNhzkaHNzyx4KGP-fFKJkS5h1AjZ9U3q1yFrqs8JdviUCIGxFiJGW86stgdT3DMLmx5rfOuOdOAhW7o0JXBN0WpuV"
[dashsegments] Total fragments: 2
[download] Destination: Planets Can Survive Being Swallowed by a Star With Surprising Effects [L7z-VeAhwCw].f140.m4a
[download] 100% of   20.00MiB in 00:00:01 at 10.35MiB/s
[Merger] Merging formats into "Planets Can Survive Being Swallowed by a Star With Surprising Effects [L7z-VeAhwCw].mp4"
[debug] ffmpeg command line: ffmpeg -y -loglevel repeat+info -i 'file:Planets Can Survive Being Swallowed by a Star With Surprising Effects [L7z-VeAhwCw].f299.mp4' -i 'file:Planets Can Survive Being Swallowed by a Star With Surprising Effects [L7z-VeAhwCw].f140.m4a' -c copy -map 0:v:0 -map 1:a:0 -movflags +faststart 'file:Planets Can Survive Being Swallowed by a Star With Surprising Effects [L7z-VeAhwCw].temp.mp4'
[debug] ffmpeg version N-109989-g7a8560cb22-20230310 Copyright (c) 2000-2023 the FFmpeg developers
  built with gcc 12.2.0 (crosstool-NG 1.25.0.90_cf9beb1)
  configuration: --prefix=/ffbuild/prefix --pkg-config-flags=--static --pkg-config=pkg-config --cross-prefix=x86_64-ffbuild-linux-gnu- --arch=x86_64 --target-os=linux --enable-gpl --enable-version3 --disable-debug --enable-iconv --enable-libxml2 --enable-zlib --enable-libfreetype --enable-libfribidi --enable-gmp --enable-lzma --enable-fontconfig --enable-libvorbis --enable-opencl --enable-libpulse --enable-libvmaf --enable-libxcb --enable-xlib --enable-amf --enable-libaom --enable-libaribb24 --enable-avisynth --enable-chromaprint --enable-libdav1d --enable-libdavs2 --disable-libfdk-aac --enable-ffnvcodec --enable-cuda-llvm --enable-frei0r --enable-libgme --enable-libkvazaar --enable-libass --enable-libbluray --enable-libjxl --enable-libmp3lame --enable-libopus --enable-mbedtls --enable-librist --enable-libssh --enable-libtheora --enable-libvpx --enable-libwebp --enable-lv2 --disable-libmfx --enable-libvpl --enable-openal --enable-libopencore-amrnb --enable-libopencore-amrwb --enable-libopenh264 --enable-libopenjpeg --enable-libopenmpt --enable-librav1e --enable-librubberband --disable-schannel --enable-sdl2 --enable-libsoxr --enable-libsrt --enable-libsvtav1 --enable-libtwolame --enable-libuavs3d --enable-libdrm --enable-vaapi --enable-libvidstab --enable-vulkan --enable-libshaderc --enable-libplacebo --enable-libx264 --enable-libx265 --enable-libxavs2 --enable-libxvid --enable-libzimg --enable-libzvbi --extra-cflags=-DLIBTWOLAME_STATIC --extra-cxxflags= --extra-ldflags=-pthread --extra-ldexeflags=-pie --extra-libs='-ldl -lgomp' --extra-version=20230310
  libavutil      58.  3.100 / 58.  3.100
  libavcodec     60.  6.101 / 60.  6.101
  libavformat    60.  4.100 / 60.  4.100
  libavdevice    60.  2.100 / 60.  2.100
  libavfilter     9.  4.100 /  9.  4.100
  libswscale      7.  2.100 /  7.  2.100
  libswresample   4. 11.100 /  4. 11.100
  libpostproc    57.  2.100 / 57.  2.100
Input #0, matroska,webm, from 'file:Planets Can Survive Being Swallowed by a Star With Surprising Effects [L7z-VeAhwCw].f299.mp4':
  Metadata:
    encoder         : google/video-file
  Duration: 00:11:20.47, start: 0.000000, bitrate: 617 kb/s
  Stream #0:0(eng): Video: vp9 (Profile 0), yuv420p(tv, bt709), 1280x720, SAR 1:1 DAR 16:9, 30 fps, 30 tbr, 1k tbn (default)
Input #1, matroska,webm, from 'file:Planets Can Survive Being Swallowed by a Star With Surprising Effects [L7z-VeAhwCw].f140.m4a':
  Metadata:
    encoder         : google/video-file
  Duration: 00:11:20.47, start: 0.000000, bitrate: 246 kb/s
  Stream #1:0(eng): Video: vp9 (Profile 0), yuv420p(tv, bt709), 1280x720, SAR 1:1 DAR 16:9, 30 fps, 30 tbr, 1k tbn (default)
Stream map '1:a:0' matches no streams.
To ignore this, add a trailing '?' to the map.

ERROR: Postprocessing: To ignore this, add a trailing '?' to the map.
Traceback (most recent call last):
  File "/usr/local/bin/yt-dlp/yt_dlp/YoutubeDL.py", line 3323, in process_info
    replace_info_dict(self.post_process(dl_filename, info_dict, files_to_move))
  File "/usr/local/bin/yt-dlp/yt_dlp/YoutubeDL.py", line 3503, in post_process
    info = self.run_all_pps('post_process', info, additional_pps=info.get('__postprocessors'))
  File "/usr/local/bin/yt-dlp/yt_dlp/YoutubeDL.py", line 3485, in run_all_pps
    info = self.run_pp(pp, info)
  File "/usr/local/bin/yt-dlp/yt_dlp/YoutubeDL.py", line 3463, in run_pp
    files_to_delete, infodict = pp.run(infodict)
  File "/usr/local/bin/yt-dlp/yt_dlp/postprocessor/common.py", line 24, in run
    ret = func(self, info, *args, **kwargs)
  File "/usr/local/bin/yt-dlp/yt_dlp/postprocessor/common.py", line 129, in wrapper
    return func(self, info)
  File "/usr/local/bin/yt-dlp/yt_dlp/postprocessor/ffmpeg.py", line 837, in run
    self.run_ffmpeg_multiple_files(info['__files_to_merge'], temp_filename, args)
  File "/usr/local/bin/yt-dlp/yt_dlp/postprocessor/ffmpeg.py", line 329, in run_ffmpeg_multiple_files
    return self.real_run_ffmpeg(
  File "/usr/local/bin/yt-dlp/yt_dlp/postprocessor/ffmpeg.py", line 367, in real_run_ffmpeg
    raise FFmpegPostProcessorError(stderr.strip().splitlines()[-1])
yt_dlp.postprocessor.ffmpeg.FFmpegPostProcessorError: To ignore this, add a trailing '?' to the map.
```

**Comment by nicolaasjan** [comment]
The same video downloads fine with youtube-dl.

**Comment by pukkandan** [comment]
**off topic**:

> [debug] yt-dlp version nightly@2023.03.11 [2a23d92d9] (zip)

This looks like a bug in build workflow @bashonly @Grub4K

**Comment by pukkandan** [comment]
> [debug] yt-dlp version nightly@2023.03.11 [2a23d92d9] (zip)

Is this a custom build, @nicolaasjan ?

**Comment by nicolaasjan** [comment]
> Is this a custom build, @nicolaasjan ?

Yes, locally compiled yesterday (with `stable` changed to `nightly` in `version.py`).
Same happened with a Windows build.

**Comment by pukkandan** [comment]
It has nothing to do with the issue. I was just confused by the version number

**Comment by nicolaasjan** [comment]
Well, e389d17 seems to have fixed it. 🙂

Can you ELI5 what happened?

**Comment by pukkandan** [comment]
The previous code was like:
```py
dct = {...}  # f1
f1_fragments = LazyList(.. dct ..)
dct = {...}  # f2
f2_fragments = ...
...
```
The `.. dct ..` part is a generator and is expanded only when the LazyList is accessed. At that point, `dct` is already of `f2` and so contains the wrong data. Now the scoping issue is avoided by putting it in a function and explicitly passing the dict to it (`f1_fragments = build_fragments(dct)`)
